src/components/LandPage.js

```diff
@@ -0,0 +1,75 @@
+import {
+  HeaderStyle,
+  LandPageStyle,
+  SearchBoxStyle,
+} from "../styles/LandPage.styled";
+import { useContext, useEffect } from "react";
+import Context from "./context";
+import MoviesContainer from "./MoviesContainer";
+import NoResult from "./NoResult";
+import { useNavigate } from "react-router-dom";
+
+export const LandPage = () => {
+  let navigate = useNavigate();
+  const a = useContext(Context);
+
+  const API_KEY = "api_key=0decbb49ed9b3d0c6017d7721a14c106";
```

Comment:
Move this private/sensitive data inside .env file. Please refer this doc: https://create-react-app.dev/docs/adding-custom-environment-variables/